src/layers/recurrent.jl

```diff
@@ -636,3 +636,67 @@ function Base.show(io::IO, g::GRUCell{use_bias, TS}) where {use_bias, TS}
     TS && print(io, ", train_state=true")
     return print(io, ")")
 end
+
+"""
+    BidirectionalRNN(cell::AbstractRecurrentCell;
+        backward_cell::Union{AbstractRecurrentCell, Nothing}=nothing,
+        merge_mode::Union{Function, Nothing}=vcat,
+        ordering::AbstractTimeSeriesDataBatchOrdering=BatchLastIndex())
+
+Bidirectional RNN wrapper.
+
+## Arguments
+
+  - `cell`: A recurrent cell. See [`RNNCell`](@ref), [`LSTMCell`](@ref), [`GRUCell`](@ref),
+    for how the inputs/outputs of a recurrent cell must be structured.
+
+## Keyword Arguments
+
+  - `backward_cell`: A optional backward recurrent cell. If `backward_cell` is `nothing`,
+    the rnn layer instance passed as the `cell` argument will be used to generate the
+    backward layer automatically. `in_dims` of `backward_cell` should be consistent with
+    `in_dims` of `cell`
+  - `merge_mode`: Function by which outputs of the forward and backward RNNs will be combined.
+    default value is `vcat`. If `nothing`, the outputs will not be combined.
+  - `ordering`: The ordering of the batch and time dimensions in the input. Defaults to
+    `BatchLastIndex()`. Alternatively can be set to `TimeLastIndex()`.
+
+## Inputs
+
+  - If `x` is a
+
+      + Tuple or Vector: Each element is fed to the `cell` sequentially.
+
+      + Array (except a Vector): It is spliced along the penultimate dimension and each
+        slice is fed to the `cell` sequentially.
+
+## Returns
+
+  - Merged output of the `cell` and `backward_cell` for the entire sequence.
+  - Update state of the `cell` and `backward_cell`.
+
+## Parameters
+
+  - `NamedTuple` with `cell` and `backward_cell`.
+
+## States
+
+  - Same as `cell` and `backward_cell`.
+"""
+struct BidirectionalRNN <: AbstractExplicitContainerLayer{(:model,)}
+    model::Parallel
+end
+
+(rnn::BidirectionalRNN)(x, ps, st::NamedTuple) = rnn.model(x, ps, st)
```

Comment:
@avik-pal Hi!  `JET.jl` `@report_opt bi_rnn(x, ps, st)` report:
``` shell
(::BidirectionalRNN)(x::Array{…}, ps::@NamedTuple{…}, st::@NamedTuple{…}) @ Lux ./Lux.jl/src/layers/recurrent.jl:690
│ runtime dispatch detected: %1::Parallel(x::Array{Float32, 3}, ps::@NamedTuple{layer_1::@NamedTuple{…}, layer_2::@NamedTuple{…}}, st::@NamedTuple{layer_1::@NamedTuple{…}, layer_2::@NamedTuple{…}})::Any
```

I have no idea why it say ` runtime dispatch ` here... Chain has same function signature `(c::Chain)(x, ps, st::NamedTuple) = applychain(c.layers, x, ps, st)`

Reply:
It should have more information in the report

Reply:
🥲 I'm sorry, it has only two lines report, like I post above